BEST_PRACTICES.md

```diff
@@ -290,68 +292,15 @@ schema:
       is not up to the user to select it.
     * `"unsupported"` means that the field exists in the source data but the
       Tap is unable to provide it.
-* `selected`: Either `true` or `false`. For a top-level schema, `true`
-   indicates that the stream should be synced, and `false` indicates it
-   should be omitted entirely. For a property within a stream, `true`
-   means include the property, `false` means leave it out.
-
-Here's an example of a discovered catalog
 
-```javascript
-{
-  "streams": [
-    {
-      "stream": "users",
-      "tap_stream_id": "users",
-      "schema": {
-        "type": "object",
-        "properties": {
-          "id": {
-            "type": "integer",
-            "inclusion": "automatic"
-          },
-          "name": {
-            "type": "object",
-            "inclusion": "available",
-            "properties": {
-              "first_name": {"type": "string", "inclusion": "available"},
-              "last_name": {"type": "string", "inclusion": "available"}
-            },
-          },
-          "addresses": {
-            "type": "array",
-            "inclusion": "available",
-            "items": {
-              "type": "object",
-              "inclusion": "available",
-              "properties": {
-                "addr1": {"type": "string", "inclusion": "available"},
-                "addr2": {"type": "string", "inclusion": "available"},
-                "city": {"type": "string", "inclusion": "available"},
-                "state": {"type": "string", "inclusion": "available"},
-                "zip": {"type": "string", "inclusion": "available"},
-              }
-            }
-          }
-        }
-      }
-    },
-    {
-      "stream": "orders",
-      "tap_stream_id": "orders",
-      "schema": {
-        "type": "object",
-        "properties": {
-          "id": {"type": "integer"},
-          "user_id": {"type": "integer", "inclusion": "available"},
-          "amount": {"type": "number", "inclusion": "available"},
-          "credit_card_number": {"type": "string", "inclusion": "available"},
-        }
-      }
-    }
-  ]
-}
+Each piece of metadata has the following canonical shape:
 ```
+{'metadata' : { 'selected' : True, 'some-other-metadata' : 'whatever' }, 'breadcrumb' : ['properties', 'some-field-name']}
+```
```

Comment:
This would read better if you indented it and used `json` after the triple-backtick.